### PR TITLE
Revise Clauses Section of Documentation

### PR DESCRIPTION
**Pull request by dehowef**
Edits the clauses section of the documentation for content, spelling, grammar, and other improvements.

### CODE REVIEW

File: docs/clauses/match.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 # MATCH
 
-The `MATCH` clause allows you to specify the patterns Cypher will search for in the database. This is the primary way of getting data into the current set of bindings. It is worth reading up more on the specification of the patterns themselves in Patterns.
+The `MATCH` clause allows you to specify the patterns a query will search for in the database. This is the primary way of retrieving data for use in a query.
 
-`MATCH` is often coupled to a `WHERE` part which adds restrictions, or predicates, to the `MATCH` patterns, making them more specific. The predicates are part of the pattern description, and should not be considered a filter applied only after the matching is done. This means that `WHERE` should always be put together with the `MATCH` clause it belongs to.
+ A `WHERE` clause often follows a `MATCH` clause to add user-defined constraints to the matched patterns to manipulate the set of data returned. The constraint predicates are part of the pattern description, and should not be considered a filter applied only after the matching is done. This means that `WHERE` should always be put together with the `MATCH` clause it belongs to.
```
Comment (rafsun42) [inline docs/clauses/match.md:5]: Looks good! Just one thought. Don't we also use the term 'constraint' to refer to database constraints like NOT NULL, DEFAULT etc? 🤔

Comment (dehowef) [reply docs/clauses/match.md]: Reverted~

File: docs/clauses/set.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ Result
 
 ## Remove a property
 
-Normally you remove a property by using `REMOVE`, but it’s sometimes handy to do it using the `SET` command. One example is if the property comes from a parameter.
+Normally a property can be removed by using `REMOVE`, but it’s users can also remove properties using the `SET` command. One example is if the property comes from a parameter.
```
Comment (jrgemignani) [inline docs/clauses/set.md:86]: Wrong "it's"?

File: docs/clauses/match.md [hunk 3]
```diff
@@ -184,7 +184,7 @@ Returns any vertices connected with the `Person` 'Oliver' that are labeled `Movi
 
 ### Outgoing Edges
 
-When the direction of an edge is of interest, it is shown by using `->` or `<-`.
+To a return directed edges, you may use `->` or `<-` to specify the direction of which the edge points.
```
Comment (jrgemignani) [inline docs/clauses/match.md:187]: I think you meant, "To return,..."

File: docs/clauses/match.md [hunk 4]
```diff
@@ -340,7 +340,7 @@ Returns `ACTED_IN` roles for 'Wall Street'.
 
 ### Multiple Edges
 
-Edges can be expressed by using multiple statements in the form of `()-[]-()`, or they can be strung together.
+Edges can be expressed by strung together to match an infinite number of edges. As long as the base pattern `()-[]-()` is followed, users can chain together edges and vertices to match specific patterns.
```
Comment (jrgemignani) [inline docs/clauses/match.md:343]: Not sure you meant to keep in "strung". Maybe stringing?

**Review by jrgemignani — CHANGES_REQUESTED** [review]

**Comment by dehowef** [comment]
@jrgemignani Good catches, thanks for looking over it. I gave those files a once over again as well, changed a few more things and also addressed all the changes you brought up

**Pull request merged**